### ISSUE TITLE
Enhancement - Print Bottle Sorting Labels by Table

### ISSUE THREAD

**Issue by macewank** [geoffhumphrey/brewcompetitiononlineentry#1158]
Version: 2.1.18

Installation URL:  forwhatitswort.brewcomp.com

Is your installation hosted on brewcompetition.com or brewcomp.com?  brewcomp.com

Description of Issue:   it would be helpful to me if we were able to sort the printed bottle sorting labels based on the table they're assigned to in the app vs based on the numerically-sorted category.   our competition often takes beers from "Category 22" and splits them into multiple tables, and when we're organizing it's difficult to keep track of who has what.  

Not a huge deal, but something I was thinking about last year when we sorted bottles and something I'm thinking about again this year.. 

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by ercousin** [comment]
This sounds like a duplicate of #1001 to me. No?

**Comment by macewank** [comment]
Not necessarily, no.  I'm speaking of the sorting labels which aren't addressed in #1001 .   Though, I suppose one could argue "being able to print all of the labels by table" broadly addresses both.
